presto-main/src/main/java/com/facebook/presto/sql/planner/ExpressionInterpreter.java

```diff
@@ -142,6 +144,7 @@
 @Deprecated
 public class ExpressionInterpreter
 {
+    private static final long MAX_SERIALIZABLE_OBJECT_SIZE = 1000;
```

Comment:
Let's do some experiment to see whether any existing scheduled query would hit this.

Reply:
Yes, will run an explain verification.